src/data/faq.json

```
@@ -2035,6 +2035,41 @@
     },
     "glossary": {
         "glossaryHeadline": "Glossary",
+        "optionalText": "",
+        "glossaryWords": {
+            "B": [
+                {
+                    "term": "Business rules",
+                    "description": "For the introduction of digital vaccination certificates, the eHealth network of the European Comission and the EU member states reconciled that every country compiles the individual rules which have to be checked for transnational travelling. These \"business rules\" have two categories:<ul><li>Invalidation rules of the country of administration, which decide under which circumstances a certificate is rejected</li><li>Acceptance rules of the country of arrival, which must be fulfilled to enter the country</li></ul>"

```

```suggestion
                    "description": "For the introduction of digital vaccination certificates, the eHealth network of the European Commission and the EU member states agreed that every country compiles the individual rules which have to be checked for transnational travelling. These \"business rules\" have two categories:<ul><li>Invalidation rules of the country of administration, which decide under which circumstances a certificate is rejected</li><li>Acceptance rules of the country of arrival, which must be fulfilled to enter the country</li></ul>"
```
Typo and better translation for "abgestimmt".

---

Also as suggestions for capitalization, referring to how the document
https://ec.europa.eu/health/sites/default/files/ehealth/docs/eu-dcc_validation-rules_en.pdf writes certain terms:

write 
- "Business Rules"
- "Invalidation Rules"
- "Acceptance Rules"
- "Country of Administration"
- "Country of Arrival"

---

Furthermore, the German and English versions both refer only to vaccination certificates whereas the DCC covers vaccination, test and recovery certificates, and the Business Rules will be applied to all three types. So although the initial certificates were only for vaccination it may be confusing here in the glossary not to refer to the other types of certificate.